guides/source/layouts_and_rendering.md

```diff
@@ -238,7 +238,7 @@ TIP: This is useful when you're rendering a small snippet of HTML code.
 However, you might want to consider moving it to a template file if the markup
 is complex.
 
-NOTE: This option will escape HTML entities if the string is not HTML safe.
+NOTE: When using `html:` option, HTML entities will be escaped if the string is not marked as HTML safe by using `html_safe` method.  
```

Comment:
Although its hard to determine what needs to be replaced. `html_safe` has its own section defined https://github.com/rails/rails/blob/3d1f95d0e8ca70f5d892d9b2ed78e99ff7cb3bde/guides/source/active_support_core_extensions.md#safe-strings with output_safety.

So only replace uses of `html_safe` in `AV` ?